### PR TITLE
Remove old machine view

### PR DESCRIPTION
**Pull request by huwshimi**
Remove all code and assets for the old machine view.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- clean and build the gui
- run the gui and check there are no errors

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4221/
Test PASSed.

File: config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ var config = {
     'scss_dirs': [
       // Must have the top level path first.
       __dirname + '/jujugui/static/gui/src/app/assets/css/',
-      __dirname + '/jujugui/static/gui/src/app/assets/css/machine-view'
+      __dirname + '/jujugui/static/gui/src/app/assets/css/canvas'
```
Comment (hatched) [inline config.js]: Do we still use this file? Why the change to 'canvas'?

Comment (huwshimi) [inline config.js]: I'm not sure if we still use it. I know in the past if a dir wasn't here the css would not be watched for changes and at some point I added a canvas dir and forgot to add it here which I noticed when removing the machine-view dir.

**Comment by hatched** [comment]
:+1: :ok: Ahh this is awesome!
The new code is 4990 total lines including tests and scss so this is a huge savings!

**Comment by kadams54** [comment]
QA is :ok: and code is :+1: for me as well. Nice!

**QA Notes:** I spun up a local env to test and attempted a variety of MV operations: scaling up, scaling down, autoplacing units, manually placing them, adding them to new containers or machines, removing units/containers/machines, sorting, etc.

**Comment by huwshimi** [comment]
Thanks @hatched and @kadams54 :shipit:!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
